### DISCUSSION TITLE
finetune with fparam

### DISCUSSION THREAD

**Discussion by chtchelkatchev** [deepmodeling/deepmd-kit#3255] [General]
I pretrained the deepmd potential with fparam using the descriptor se_atten_v2 and deepmd=2.2.7. Then an attempt was made to finetune this potential on a new database with fparam, which led to an error message: "... in _prepare_feed_dict assert fparam is not None  ... AssertionError". Without fparam fintuning works well. The ussue appears only with fparam. Does finetuning work with fparam?

**Comment by njzjz** [comment]
This is a bug. I'll convert the discussion to an issue to track it.

https://github.com/deepmodeling/deepmd-kit/blob/b875ea8f6661b6e1567537ead7e2b4a8b14ea113/deepmd/fit/ener.py#L853

**Reply by chtchelkatchev** [reply]
Thank You for quick reply!
